__tests__/utils/flattenProperties.test.js

```diff
@@ -43,9 +44,9 @@ describe('utils', () => {
         properties.white
       ];
 
-      var sortedExpectedRet = _.sortBy(expected_ret, ['value']);
+      var sortedExpectedRet = expected_ret.concat().sort(sortBy("value"));
```

Comment:
Why are we doing a concat before the sort?

Reply:
Sorry for the delay. Answered below.